app/src/main/java/com/example/una/adapters/VerticalRecyclerViewAdapter.java

```diff
@@ -40,7 +40,7 @@ public void onBindViewHolder(@NonNull VerticalRVViewHolder holder, int index) {
         ArrayList<Object> currentItem = section.getArrayList();
         RecyclerView.Adapter recyclerViewAdapter = null;
 
-        if (holder.getItemViewType() == HomeFragmentSection.CHARITY_LIST_TYPE) {
+        if (holder.getItemViewType() == HomeFragmentSection.CHARITY_LIST_TYPE || holder.getItemViewType() == HomeFragmentSection.RECOMMENDED_LIST_TYPE) {
             recyclerViewAdapter = new CharityAdapter(currentItem);
```

Comment:
I dont think you need to add a new type. You can just use the same type that you used before because they are both the "charity list type"

Reply:
👍